### ISSUE TITLE
Feature: Add badges to ReadMe 

### ISSUE THREAD

**Issue by PCain02** [GatorEducator/execexam#16]
Description: The current version of execexam does not have any badges indicating a passing build. Additionally, it does not have any badges that indicate the PyPI version. 

Proposed Solution: Implement badges that display useful information to the viewer at a glance. Research the different types of badges available and pick the best ones for the project. Update the build.yml file to create and update these badges.

**Comment by gkapfham** [comment]
Hi @Chezka109 can you please give a status update on your work towards finishing this issue.

**Comment by Chezka109** [comment]
Progress: I have successfully implemented a dynamic coverage badge for the execexam project. The badge color now changes based on the current coverage percentage, providing an immediate visual cue on the quality of the code coverage. Additionally, I have added a maintenance badge to reflect the project's maintenance status, ensuring that users are aware of ongoing upkeep efforts.

Next Steps: I plan to add more badges, including a badge indicating the PyPI version of the project, as well as a passing build badge. These will give users and contributors quick, at-a-glance insights into the state of the project.
